inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -76,6 +77,7 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     manager.register_pass<ngraph::pass::HSigmoidFusion>();
     manager.register_pass<ngraph::pass::ConvertPadToGroupConvolution, false>();
     manager.register_pass<ngraph::pass::NormalizeL2Fusion>();
+    manager.register_pass<ngraph::pass::ReluFakeQuantizeFusion>();
```

Comment:
I think we have a better place for this fusion: https://github.com/openvinotoolkit/openvino/pull/3811/files#diff-7b69763d5d7d534dfcbe73d43d300bbe45b8aee612400686591fb88f4cd9adecR120
Just add it to the group of `fq_fusions ` transformations.